chcemvediet/apps/inforequests/admin.py

```diff
@@ -253,13 +253,51 @@ class InforequestEmailAdmin(admin.ModelAdmin):
             ]
     inlines = [
             ]
+    actions = [
+            u'delete_selected'
+    ]
 
     def get_queryset(self, request):
         queryset = super(InforequestEmailAdmin, self).get_queryset(request)
         queryset = queryset.select_related(u'inforequest')
         queryset = queryset.select_related(u'email')
         return queryset
 
+    def delete_constraints(self, objs):
+        constraints = []
+        for obj in objs:
+            if hasattr(obj.email, u'action'):
+                if obj.inforequest == obj.email.action.branch.inforequest:
+                    constraints.append(format_html(
+                        u'{} is used in its inforequest.'.format(admin_obj_format(obj))))
```

Comment:
Skor by som to naformuloval nejak takto:

> '{} is used in {}'.format(obj, action)

Reply:
:ok: